lineapy/api/api_classes.py

```diff
@@ -156,7 +158,15 @@ def get_code(
         )
         if not use_lineapy_serialization:
             code = de_lineate_code(code, self.db)
-        return prettify(code)
+
+        pretty_code = prettify(code)
+        if new_cell:
```

Comment:
Do not use implicit type conversion. This has caused issues in the past with lineapy versions. Instead use `if new_cell is not None`.